articles/migrations/index.md

```diff
@@ -25,6 +25,85 @@ If you need help with the migration, create a ticket in our [Support Center](${e
+
+We’ve introduced support for Resource Owner Password to the `oauth/token` endpoint earlier this year ([doc](/api/authentication#resource-owner-password)). We will deprecate the current `oauth/ro` and `oauth/access_token` endpoints. Applications must be updated before July 1, 2018, when `oauth/ro` will become unavailable. A migration guide will be available in Q1.
+
+#### Am I affected by the change?
+
+If you are currently implementing the `oauth/ro` endpoint your application will need to be updated.
+
+If you have any questions, create a ticket in our [Support Center](${env.DOMAIN_URL_SUPPORT}).
+
+### API Authorization with 3rd Party Vendor APIs. Deprecating Delegation Endpoint
+
+| Severity | Grace Period Start | Mandatory Opt-In|
+| --- | --- | --- |
+| Medium | 2017-12-21 |  2018-06-01 |
+
+We’re re-architecting the mechanism by which you get tokens for 3rd Party/Vendor APIs (such as AWS, Firebase, and others). It now works the same as any custom API, providing better consistency. This new architecture will be available in Q1 2018 and its availability will officially deprecate the Delegation Endpoint. All clients must be updated prior to June 1, 2018. Formal migration guide will be available in Q1.
+
+#### Am I affected by the change?
+
+If you are currently using delegation to provide third party authorization, your application will need to be updated.
+
+If you have any questions, create a ticket in our [Support Center](${env.DOMAIN_URL_SUPPORT}).
+
+### Deprecating the Usage of ID Tokens on the Auth0 Management API
+
+| Severity | Grace Period Start | Mandatory Opt-In|
+| --- | --- | --- |
+| Medium | 2017-12-21 |  2018-06-01 |
+
+We are deprecating the usage of id tokens when calling `PATCH/GET` `/user` and `/device-credentials`. We have moved to regular access tokens, and this is available now. Applications must be updated by June 1, 2018, when the ability to use id tokens will become unavailable. Migration guides will be available by February 2018.
+
+#### Am I affected by the change?
+
+If you are currently using id tokens to access any part of the Management API, your application will need to be updated.
+
+If you have any questions, create a ticket in our [Support Center](${env.DOMAIN_URL_SUPPORT}).
+
+### Improved OpenID Connect Interoperability in Auth0
+
+| Severity | Grace Period Start | Mandatory Opt-In|
+| --- | --- | --- |
+| Medium | 2017-12-21 |  2018-07-01 |
+
+We’re updating the `userinfo` endpoint to return OIDC conformant user profile attributes. The most notable change is that `user_id` becomes `sub`. This will deprecate the legacy Auth0 user profile (in `userinfo` and in ID Tokens). Applications must be updated to use the new user profile before July 1, 2018. Migration guide will be available in late Q1 2018.
+
+#### Am I affected by the change?
+
+If you are currently using the `userinfo` endpoint to return user information, you will need to ensure that your applications are migrated where applicable.
```

Comment:
- link to endpoint
- `to return` -> `to retrieve` or `to get`
- `you will need to ensure that your applications are migrated where applicable.` -> `you are affected by this change and need to update your implementation`.